Apps/frontend/src/labels.js

```diff
@@ -10,6 +10,12 @@ export const TOOLTIP_BUTTON_OFFOFF_ON = "Turn on";
 export const TOOLTIP_BUTTON_OFFOFF_OFF = "Turn off";
 export const TOOLTIP_BUTTON_SETTINGS = "Open settings";
 
+export const TOOLTIP_CONTROL_HEADER_ONOFF = "Can be used to switch on/off particular channels"
+export const TOOLTIP_CONTROL_HEADER_THICKNESS = "Can be used to make particular channels thick"
+export const TOOLTIP_CONTROL_HEADER_OFFSET = "Can be used to control the offset to the zero-line"
+export const TOOLTIP_CONTROL_HEADER_TIMESWEEP = "Controls the time-sweep for a perticular channel"
+export const TOOLTIP_CONTROL_HEADER_AMPLITUDE = "Controls (scales) the amplitudes of the signals"
+
```

Comment:
Please run `npm run lint` again. The lines are too long. 
![image](https://user-images.githubusercontent.com/13545730/215324681-196ad877-83ab-41e0-a116-e6b10ba7132a.png)